README.md

```diff
@@ -65,7 +65,7 @@ Replace `<GITHUB_TOKEN>`, `<OPENAI_TOKEN>`, `<OWNER>`, `<REPO>`, and `<PR_NUMBER
```

Comment:
It is unclear what the `jira_url` parameter is for. More documentation would be helpful.